.kitchen.yml

```diff
@@ -60,6 +64,31 @@ suites:
           checkpoint: true
           switch_user: yes
 
+- name: attributes
```

Comment:
So, I've checked a bit YAML anchors and it seems it is not that great as YAML doesn't do a recursive hash merge, so for each level one would want to change a new template would have to be defined (come to think of it, this is probably the reason I have not seen any YAML anchors in kitchen tests in general).